Sources/SwiftProtobuf/JSONEncodingOptions.swift

```diff
@@ -15,6 +15,10 @@
 /// Options for JSONEncoding.
 public struct JSONEncodingOptions {
 
+  /// Always prints int64s as numbers.
+  /// By default, they are printed as strings as per proto3 JSON mapping rules.
```

Comment:
We should clarify here that they are still printed as strings when used as Map keys.

Reply:
Comment updated. Let me know if this is ok.